Documentation/BasicUsage.md

```diff
@@ -0,0 +1,267 @@
+# Basic Usage
+
+For starters we're going to import everything ready to use.  
+Most of the included functionality is similar to how you would deal with WebRTC in your browser.  
+We support most of the official WebRTC APIs, see this [document](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) for more details.
+
+```javascript
+import {
+	ScreenCapturePickerView,
+	RTCPeerConnection,
+	RTCIceCandidate,
+	RTCSessionDescription,
+	RTCView,
+	MediaStream,
+	MediaStreamTrack,
+	mediaDevices,
+	permissions,
+	registerGlobals
+} from 'react-native-webrtc';
+```
+
+## Defining Media Constraints
+
+```javascript
+var mediaConstraints = {
+	audio: {
+		googEchoCancellation: true,
+		googAutoGainControl: true,
+		googNoiseSuppression: true,
+		googHighpassFilter: true,
+		googEchoCancellation2: true,
+		googAutoGainControl2: true,
+		googNoiseSuppression2: true
+	},
+	video: {
+		mandatory: {
+			minFrameRate: '30'
+		},
+		facingMode: 'user',
+		optional: []
+	}
+};
+```
+
+## Getting a Media Stream
+
+```javascript
+var localMediaStream;
+var isVoiceOnly: boolean = false;
+
+try {
+	const mediaStream = await mediaDevices.getUserMedia( mediaConstraints );
+
+	if ( isVoiceOnly ) {
+		let videoTrack = await mediaStream.getVideoTracks()[ 0 ];
+		videoTrack.enabled = false;
+	};
+
+	localMediaStream = mediaStream;
+} catch( err ) {
+	// Handle Error
+};
+```
+
+Promise based method.
```

Comment:
I find this second example confusing. Both are promise based, but the former uses the async / await syntax. I'd just leave that one.

Reply:
Ahahaha yeah i haven't thought of how i'd display Async/Await and raw Promise methods yet.
Any suggestions?

Reply:
I think everyone is familiar with async / await now. I think it also encourages better error handling.

Reply:
Done in the next commit coming later today.